layouts/partials/ecosystem/registry/entry.html

```diff
@@ -141,6 +141,19 @@ <h4 class="alert-heading">Deprecated</h4>
                     {{ end -}}
                     {{- .description | markdownify -}}
                 </div>
+
+                <!-- New Tags section -->
+                <div class="tags mb-2">
+                    {{ if .tags }}
+                        <div>
+                            {{ range .tags }}
+                                {{ $tag := . }} <!-- Store the tag in a variable -->
+                                <a href="/search?query={{ $tag | urlQuery }}" class="badge bg-light me-1">{{ $tag }}</a>
```

Comment:
urlQuery is not a function, you might be looking for `anchorize` or `urlize`:

https://gohugo.io/functions/urls/anchorize/

Reply:
@svrnm Done sir...thanks for pointing that out